python/lsst/rucio/register/script.py

```diff
@@ -91,9 +94,19 @@ def _register(ri, dataset_refs, chunk_size, rucio_dataset):
         logger.debug(f"{cnt} butler datasets registered")
 
 
-def _set_log_level(debug):
+def _register_zips(ri, zip_files, chunk_size, rucio_dataset):
+    # register dataset_refs with Rucio into the rucio dataset, in chunks
+    for zip_file in zip_files:
+        rp = ResourcePath(zip_file)
+        cnt = ri.register_zips(rucio_dataset, [rp])
+        logger.debug(f"{cnt} zips registered")
+
+
+def _set_log_level(debug, info):
```

Comment:
It's a private function so feel free to ignore this comment, but I think something like `_set_logging()` would be more appropriate name for it as it customizes more than just the log level.

Reply:
I left this as is because of the change I made to the command line option.